### DISCUSSION TITLE
How to add SCSS in tailwindcss project?

### DISCUSSION THREAD

**Discussion by azimuddin017** [community/community#67945] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How to add SCSS in tailwindcss project?

**Comment by web-emran** [comment]
You can follow this doc https://tailwindcss.com/docs/guides/nextjs

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
